changelogs/current.yaml

```
@@ -16,5 +16,9 @@ new_features:
 - area: extension_discovery_service
   change: |
     added ECDS support for :ref:` downstream network filters<envoy_v3_api_field_config.listener.v3.Filter.config_discovery>`.
+- area: tls
+  change: |
+    Added new metric for emitting seconds until expirations of TLS & CA certificates whcih are rooted at the cert.<CERT_NAME>. namespace.
```

```suggestion
    Added new metric for emitting seconds until expirations of TLS & CA certificates which are rooted at the cert.<CERT_NAME>. namespace.
```

Also, I don't think that's the right namespace anymore

---

also updated docs